core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.spec.ts

```diff
@@ -1,22 +1,267 @@
+import { DotActionButtonModule } from '@components/_common/dot-action-button/dot-action-button.module';
+import { DotActionMenuButtonModule } from '@components/_common/dot-action-menu-button/dot-action-menu-button.module';
+import { DotAddToBundleModule } from '@components/_common/dot-add-to-bundle';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { CONTAINER_SOURCE, DotContainer } from '@models/container/dot-container.model';
+
+const containersMock: DotContainer[] = [
+    {
+        archived: false,
+        categoryId: '6e07301c-e6d2-4c1f-9e8e-fcc4a31947d3',
+        deleted: false,
+        friendlyName: '',
+        identifier: 'f17f87c0e571060732923ec92d071b73',
+        live: true,
+        name: 'movie',
+        parentPermissionable: {
+            hostname: 'default'
+        },
+        path: null,
+        source: CONTAINER_SOURCE.DB,
+        title: 'movie',
+        type: 'containers',
+        working: true
+    },
+    {
+        archived: false,
+        categoryId: 'a443d26e-0e92-4a9e-a2ab-90a44fd1eb8d',
+        deleted: false,
+        friendlyName: '',
+        identifier: '282685c94eb370a7820766d6aa1d0136',
+        live: true,
+        name: 'test',
+        parentPermissionable: {
+            hostname: 'default'
+        },
+        path: null,
+        source: CONTAINER_SOURCE.DB,
+        title: 'test',
+        type: 'containers',
+        working: true
+    }
+];
+
+const columnsMock = [
+    {
+        fieldName: 'title',
+        header: 'Name',
+        sortable: true
+    },
+    {
+        fieldName: 'status',
+        header: 'Status',
+        width: '8%'
+    },
+    {
+        fieldName: 'friendlyName',
+        header: 'Description'
+    },
+    {
+        fieldName: 'modDate',
+        format: 'date',
+        header: 'Last Edit',
+        sortable: true
+    }
+];
+
+const messages = {
+    'Add-To-Bundle': 'Add To Bundle',
+    'Remote-Publish': 'Push Publish',
+    'code-container': 'Advanced Container',
+    'contenttypes.content.push_publish': 'Push Publish',
+    'design-container': 'Container Designer',
+    'message.container.confirm.delete.container':
+        'Are you sure you want to delete this Container?  (This operation cannot be undone)',
+    'message.container.copy': 'Container copied',
+    'message.container.delete': 'Container archived',
+    'message.container.full_delete': 'Container deleted',
+    'message.container.undelete': 'Container unarchived',
+    'message.container.unpublished': 'Container unpublished',
+    'message.container_list.published': 'Containers published',
+    'containers.fieldName.description': 'Description',
+    'containers.fieldName.lastEdit': 'Last Edit',
+    'containers.fieldName.name': 'Name',
```

Comment:
language keys that don't exist in https://github.com/dotCMS/core/blob/14a1349d5bc65407427be7d6ca912b98cc35b4f3/dotCMS/src/main/webapp/WEB-INF/messages/Language.properties

needs to be added